src/main/java/com/shopping_app/shoppingApp/mapping/AddressMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.shopping_app.shoppingApp.mapping;
+
+import com.shopping_app.shoppingApp.domain.Address;
+import com.shopping_app.shoppingApp.model.Request.AddressRequest;
+import com.shopping_app.shoppingApp.model.Response.AddressResponse;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.ReportingPolicy;
+
+@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.WARN)
+public interface AddressMapper {
```

Comment:
What is the purpose of adding the mapper interface?

Reply:
using this we can easily convert pojo class conversion without writing by ourself , during compile time , this interface class will implement all pojo convertion logic , this will reduce our dev time

Reply:
removed mapper functionality